tests/framework/e2e/cluster.go

```diff
@@ -19,13 +19,15 @@ import (
 	"net/url"
 	"os"
 	"path"
+	"regexp"
 	"strings"
 	"testing"
 	"time"
 
-	"go.etcd.io/etcd/server/v3/etcdserver"
 	"go.uber.org/zap"
 	"go.uber.org/zap/zaptest"
+
+	"go.etcd.io/etcd/server/v3/etcdserver"
```

Comment:
@tjungblu thanks, changes lgtm but I am curious why are you changing import order here and in other files? The current order is as per overall convention used in the project (i.e. third party packages follow internal ones..) and we should keep it that way unless I am missing anything.

Reply:
@tjungblu can you please look at my small comment above?. Your PR looks good otherwise and we can merge it.

Reply:
I won't hold the PR due to the ordering changes in the import which seems unnecessary but can be reordered separately.

Reply:
Usually we shouldn't change the existing code (including this case, the order of the import entries and the blank line) without explicit reason. 

But on the other hand, it seems that it isn't a big deal on the change in this case.  Either keeping it as it's for now or reverting the change works for me. 

I suggest to keep it as it's for now. 

Then this PR is good to go, and defer to @spzala to make the final decision and merge the PR.

Reply:
Yup, sgtm. Thanks @ahrtr